hash_practice/exercises.py

```diff
@@ -1,19 +1,50 @@
 
-def grouped_anagrams(strings):
+def grouped_anagrams(words):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n^2)

```

Comment:
When doing complexity calculations, make sure that you don't reuse variables to represent two different factors.

I'm assuming that you're going off the nested loop but note that the outside loop will go over every word in the list, while the inside loop will go over every character in each word. These can't both be represented with `n`, so you should probably come up with a new variable for the second one (like `m`) and then have O(n * m).